app/presenters/warning_presenter.rb

```diff
@@ -25,23 +11,28 @@ def message
       cya_page_not_present_message
   end
 
+  def both_pages_missing?
```

Comment:
similar to above this name is too generic. `cya_and_confirmation_missing?` maybe